pallets/connectors/src/message.rs

```diff
@@ -56,13 +60,72 @@ where
 		address: Address,
 		valid_until: Moment,
 	},
+	// Bidirectional: Domain must not accept every incoming token.
+	// Sender must ensure beforehand that the receiver will not reject
 	Transfer {
+		token: u128,
+		source_address: Address,
+		destination_address: Address,
+		amount: Balance,
+	},
+	TransferTrancheTokens {
+		domain: Domain,
+		pool_id: PoolId,
+		tranche_id: TrancheId,
+		source_address: Address,
+		destination_address: Address,
+		amount: Balance,
+	},
+	IncreaseInvestOrder {
+		pool_id: PoolId,
+		tranche_id: TrancheId,
+		address: Address,
+		token: u128,
```

Comment:
In `AddPool` we are calling it `currency` while here we call it `token`. I don't have a strong preference for either but let's agree on one of the two and make that consistent both here and on the Solidity side cc @offerijns @AStox

Reply:
`currency` is widely used in tinlake contracts to refer to stable coins, so I think we could try to keep that naming convention here as well.